### DISCUSSION TITLE
Show-install prompt

### DISCUSSION THREAD

**Discussion by ShamaliR** [PSAppDeployToolkit/PSAppDeployToolkit#1043] [Q&A] [answered]
We need the message "installation done successfully " in the local languages. As we are able to give only the custom message as of now which is displayed only in English. Kindly need help on this

**Comment by mjr4077au** [comment]
You can probably concatenate the DeploymentType string and a BalloonTip string together to achieve the result?

![image](https://github.com/user-attachments/assets/035a46e5-84be-4566-a81d-713e041f1323)

**Comment by ShamaliR** [comment]
Can you please help me how to do that i don't have the idea

**Reply by mjr4077au** [reply, marked as answer]
`Show-InstallationPrompt -Message "$configDeploymentTypeInstall $configBalloonTextComplete" -ButtonRightText OK -Icon Information -NoWait`.

**Comment by ShamaliR** [comment]
When testing the script for the psadt 4.0.5 an error pop-up appears if the user does not click the defer/continue button before the defer time reaches '0'.
